docs/section1/Rule1-3.md

```diff
@@ -0,0 +1,44 @@
+# Section 1 - Rule 1-3
+**Schema Version:** 0.0.36  
+**Mandatory Rule:** True  
+**Rule ID:** 1-3
+**Rule Description:** The Performance Cost Index-Target (PCIt) shall be calculated using the procedures defined in Section 4.2.1.1. The PCIt shall be equal to baseline building unregulated energy cost (BBUEC) + BPF x baseline building regulated energy cost (BBREC)/ BBP
```

Comment:
Rule 1-5 deals with the case where on-site renewable energy generation exceeds the thresholds defined in Section 4.2.1.1.  Are projects expected to always use the formula in this rule to generate PCIt and then subtract the 5% afterwards for rule 1-5?  We may need to clarify in the schema what calculation we expect for `performance_cost_index_target` for projects that exceed the threshold defined in 4.2.1.1

Reply:
`performance_cost_index_target` is always calculated the same way. The formula in Rule 1-5 handles verifying the PCI is at least 5% less than PCIt, excluding renewables, there is no expectation that 5% is subtracted out at any point.